src/main/kotlin/racingcar/Cars.kt

```diff
@@ -6,15 +6,19 @@ import racingcar.model.RaceCondition
 /**
  * 자동차들의 객체 관리
  * */
-class Cars(raceCondition: RaceCondition) {
-    private var _carList = (0 until raceCondition.carCount).map { Car() }.toList()
+class Cars private constructor(var _carList: List<Car>) {
+    companion object {
+        fun createCars(raceCondition: RaceCondition): Cars {
+            val carList = (0 until raceCondition.carCount).map { Car() }.toList()
```

Comment:
자바와 다르게 map까진 해주시면 되요 !
```suggestion
            val carList = (0 until raceCondition.carCount).map { Car() }
```